test/val/val_builtins_test.cpp

```diff
@@ -1200,16 +1200,6 @@ INSTANTIATE_TEST_CASE_P(
         Values(TestResult(SPV_ERROR_INVALID_DATA,
                           "to be used only with Fragment execution model"))), );
 
-INSTANTIATE_TEST_CASE_P(
```

Comment:
Can this test be salvaged? Would changing the storage class to Workgroup pass the test?